### PR TITLE
minor: move module config javadoc to examples section

### PR DESCRIPTION
**Pull request by gaurabdg**
Site screenshot:
Screenshot_20200808_014245

### CODE REVIEW

File: src/xdocs/config_blocks.xml [hunk 1]
```diff
@@ -277,6 +264,18 @@ switch (a) {
       <subsection name="Examples" id="EmptyBlock_Examples">
         <p> To configure the check: </p>
         <source>
+&lt;module name=&quot;EmptyBlock&quot;&gt;
+  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_DEFAULT&quot;/&gt;
```
Comment (romani) [inline src/xdocs/config_blocks.xml:268]: Please move it below default config/example

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java [hunk 2]
```diff
@@ -121,6 +122,11 @@
             "SuppressionXpathSingleFilter - message",
         }).collect(Collectors.toSet()));
 
+    private static final Set<String> MODIFIED_JAVADOC_FILES =
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java:125]: CHECKS_WITH_EXAMPLES_PARAGRAPH

**Comment by romani** [comment]
please create new PR to just move example below "To configure check:".

We might not need "Examples" paragraph in javadoc.

File: src/xdocs/config_blocks.xml (hunk 1 above)
Comment (gaurabdg) [reply src/xdocs/config_blocks.xml]: Done

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java (hunk 2 above)
Comment (gaurabdg) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: Removed as requested

**Comment by gaurabdg** [comment]
> please create new PR to just move example below "To configure check:".
> 
> We might not need "Examples" paragraph in javadoc as `To configure the check:` is already a marker that most Checks have.

Yes, I have removed.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 3]
```diff
@@ -47,23 +47,6 @@
  * So, if tokens=LITERAL_DEFAULT, following code will not trigger any violation,
  * as the empty block belongs to LITERAL_CASE:
  * </p>
- * <p>
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:50]: change  text above to:
```
 * NOTE: This check processes LITERAL_CASE and LITERAL_DEFAULT separately.
 * Verification empty block is done for single most nearest {@code case} or {@code default}.
```

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java [hunk 4]
```diff
@@ -115,6 +98,23 @@
...
+ * </pre>
+ * <p>
+ * Result:
+ * </p>
+ * <pre>
+ * switch (a) {
+ *   default:        // no violation for "default:" as empty block belong to "case 1:"
+ *   case 1: { }
+ * }
+ * </pre>
```
[the remark this answers is not in the document]
Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:115]: Done

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java (hunk 3 above)
Comment (gaurabdg) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java:50]: Done

**Pull request merged**